etl/RetrosheetParser.py

```diff
@@ -96,22 +83,20 @@ def parse_start(self, retro_list):
 
 
     def parse_play(self, retro_list):
-        "Parse the play type"
-        # I wonder if I am over-thinking it here by trying to read only the play lines that have a meaningful play involved.  Keep this object and its methods as more purely reading the data?
-        if retro_list[6].startswith(self.play_types):
-        	play_instance = {
-        			'inning': retro_list[1]
-        			, 'home_team': retro_list[2]
-        			, 'player_id': retro_list[3]
-        			, 'count': retro_list[4]
-        			, 'pitch_seq': retro_list[5]
-        			, 'play_event': retro_list[6]
-        			}
-        	return play_instance
+        "Parse the play type with respect to the play_by_play table"
+        play_instance = {
+                'inning': retro_list[1]
+                , 'home_team': retro_list[2]
+                , 'rts_player_id': retro_list[3]
+                , 'rts_pitch_count': retro_list[4]
+                , 'rts_pitch_seq': retro_list[5]
+                , 'play_meta': retro_list[6]
+                }
+        return play_instance
+
 
     def parse_sub(self, retro_list):
         "Parse the sub type"
-        # Note: def_position == 11: pinch hitter, def_position == 12: pinch runner
```

Comment:
Since we're not doing anything we the position type here, commenting on its meaning is superfluous.